parsing/padding.go

```diff
@@ -2,103 +2,103 @@ package parsing
 
 import (
 	"errors"
-	"github.com/filecoin-project/go-data-segment/types"
+	"github.com/filecoin-project/go-data-segment/fr32"
```

Comment:
I think `types` previously was fine, we can have more types there and now we have a repetition of the `fr32.Fr32` which is a bit ugly.

Reply:
Ahh, I see why you did it, so the Bits* are nicer.

Reply:
This works as well, I would say.